apps/api/pages/api/destination-calendars/_post.ts

```diff
@@ -91,7 +109,9 @@ async function postHandler(req: NextApiRequest) {
     parsedBody.userId = undefined;
   }
 
-  const destination_calendar = await prisma.destinationCalendar.create({ data: { ...parsedBody } });
+  const destination_calendar = await prisma.destinationCalendar.create({
+    data: { ...parsedBody, credentialId },
+  });
```

Comment:
credentialId is now part of the query, so link would be made without exposure of the credentialId, which was the case earlier.